tests/EdgeToEdge/WebRouteTestCase.php

```diff
@@ -151,7 +151,13 @@ protected function assertErrorJsonResponse( Response $response ) {
 		$responseData = $this->getJsonFromResponse( $response );
 		$this->assertArrayHasKey( 'status', $responseData );
 		$this->assertEquals( $responseData['status'], 'ERR' );
-		$this->assertArrayHasKey( 'message', $responseData );
+		$this->assertThat(
+			$responseData,
+			$this->logicalOr(
+				$this->arrayHasKey( 'message' ),
```

Comment:
`mesage` is still used in the `validate-fee` and `add-comment` JSON response. Do you think the assertion is too ugly?